packages/eslint-plugin/src/rules/member-delimiter-style.ts

```diff
@@ -21,14 +28,26 @@ type Config = BaseOptions & {
     typeLiteral?: BaseOptions;
     interface?: BaseOptions;
   };
-  multilineDetection?: 'brackets' | 'last-member';
+  multilineDetection?: string;
```

Comment:
i think this is not correct, with new implementation we are using this in one condition, and type should not be changed here

```ts
options.multilineDetection === 'last-member'
```

Reply:
Could you explain a little bit more please ? 🙂

Reply:
instead of accepting any string, we should explicitly state allowed type
```ts
  multilineDetection?: 'last-member';
```

see 
https://github.com/typescript-eslint/typescript-eslint/blob/74d7e537927643393320c48d31363b0ff013e364/packages/eslint-plugin/src/rules/member-delimiter-style.ts#L280

Reply:
You're right. I fixed it here : 14491bceec7c9514ef325c51fe4146aa636b8385